### ISSUE TITLE
Add github releases for feedstocks

### ISSUE THREAD

**Issue by ZzEeKkAa** [regro/cf-scripts#2828]
Not sure where to ask this question, but I've found https://conda-forge.org/blog/2020/11/20/anaconda-tos/ post that references to `regro/releases` which is currently unavailable. I'm wondering what happen to this repo? Are there any plans to distribute builds as releases besides uploading to anaconda servers and what are the blockers? According to the https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases#storage-and-bandwidth-quotas there is no quotas on usage, except the size limit of 2GB per file, which should cover most of the conda-forge packages. It would be possible to update quartz to host `repodata.json` and redirect all the package traffic to the corresponding files in the release section of the github repo.

**Comment by beckermr** [comment]
This activity broke GitHub's backend code and so we had to stop.

**Comment by ZzEeKkAa** [comment]
Thank you! Are there any references or status update if it was fixed?

**Comment by beckermr** [comment]
It wasn't so much of a bug but actually they didn't want us uploading that much data to a repo.

**Comment by ZzEeKkAa** [comment]
How then homebrew works and it does not the case there?
https://github.com/orgs/Homebrew/packages?ecosystem=container

![image](https://github.com/regro/cf-scripts/assets/5175807/ef7ac69d-f248-4d61-9595-cacb442fe0f0)

**Comment by beckermr** [comment]
Those are container images and that is a separate issue. What are you looking to do here?

**Comment by ZzEeKkAa** [comment]
I'm looking for an option to use conda-forge without anaconda's servers

**Comment by beckermr** [comment]
Yeah that's something that is really hard to do. The amount of bandwidth and money they spend serving conda-forge is immense. We have hundreds of millions of downloads each month. They also pay for CDN services for conda-forge and some other large channels which is a significant performance enhancement.  

We have been working with the conda community to upload artifacts to the github container site, but cannot yet point conda itself at the repo afaik. 

Also, certain parts of the ecosystem have to be distributed on anaconda.org due to licensing restrictions. So any mirror is going to have that limitation.

**Comment by ZzEeKkAa** [comment]
Thank you!

> The amount of bandwidth and money they spend serving conda-forge is immense. We have hundreds of millions of downloads each month. They also pay for CDN services for conda-forge and some other large channels which is a significant performance enhancement.

Having homebrew as an example it looks like github does not have any restrictions and may provide these capacities.. I've looked into how it works, and they are using [OCI](https://github.com/opencontainers/image-spec). It should be possible to pack conda packages with OCI (aka new conda package format) and teach `conda cli` to work with it. Then github registry will provide both hosting and open analytics on the usage... The technical challenge may be to provide different way of dependency resolution, cause there won't be `repodata.json` index.

> We have been working with the conda community to upload artifacts to the github container site, but cannot yet point conda itself at the repo afaik.

So far any references to accomplished work?

> Also, certain parts of the ecosystem have to be distributed on anaconda.org due to licensing restrictions. So any mirror is going to have that limitation.

What part of ecosystem? Is there any list to be aware of? And what kind of limitations are those?

**Comment by beckermr** [comment]
The limits in terms of licensing are nvidia / cuda and possibly intel packages.

**Comment by wolfv** [comment]
The next version of mamba should also get support for accessing the OCI registry, by the way. If you want to help us make this nice, let me know! Easiest would be to continue chatting on our Discord server (you can find it on https://prefix.dev)

**Comment by beckermr** [comment]
Have we written a plugin to conda for this yet or is there more involved there?

**Comment by wolfv** [comment]
Jannis has some plans, but I don't know if that ever progressed anywhere :) But you can use all of this very nicely from `rattler` / `py-rattler` as well.

**Comment by wolfv** [comment]
yes, we are working hard to make sure that there is no vendor-lock-in for the conda-forge community :)
